spec/lib/bundler/explain/parser_spec.rb

```diff
@@ -2,10 +2,26 @@
 
 describe Bundler::Explain::Parser do
   describe '#call' do
-    it 'process file to hash' do
-      file_path = "#{File.dirname(__FILE__)}/../../../examples/case1/Gemfile.lock"
-      result = Bundler::Explain::Parser.new(file_path).call
-      expect(result.dig('GEM', 'specs', 'actioncable (5.1.4)').keys).to match_array ['actionpack (= 5.1.4)', 'nio4r (~> 2.0)', 'websocket-driver (~> 0.6.1)']
+    let(:gemfile) { File.expand_path("#{File.dirname(__FILE__)}/../../../examples/case1/Gemfile") }
+    let(:gemfile_lock) { File.expand_path("#{File.dirname(__FILE__)}/../../../examples/case1/Gemfile.lock") }
+
+    subject { Bundler::Explain::Parser.new(gemfile: gemfile, gemfile_lock: gemfile_lock).call }
+
+    it 'processing success' do
+      expect(subject.success?).to eq true
+    end
```

Comment:
With `rspec-its` gem it can be just `its(:success?) { is_expected.to be_truthy }`. But in general, predicate-style methods can be tested this way: `it { is_expected.to be_success }` (RSpec will understand to call `success?` automatically)